src/Symfony/Component/DependencyInjection/Tests/Fixtures/php/services10.php

```diff
@@ -30,6 +30,11 @@ public function __construct()
         $this->parameters = $this->getDefaultParameters();
 
         $this->services = array();
+        $this->normalizedIds = array(
+            'psr\\container\\containerinterface' => 'Psr\\Container\\ContainerInterface',
```

Comment:
@nicolas-grekas should we really dump normalized ids for services which got removed ?